Cargo.toml

```diff
@@ -53,3 +53,8 @@ harness = false
 [[bench]]
 name = "cctv"
 harness = false
+
+[patch.crates-io]
```

Comment:
Should we wait until a pre-release version of `crypto-bigint` is pubished?

Reply:
Yeah, we can. I am trying to bump `rand_core` on `rsa` to bring `signature 2.3.0-pre.5` which bring in `rand_core` 0.9.

I'm fine either way.

Reply:
If `crypto-bigint` release is somewhere close, I'd rather wait. Depending on the trunk can lead to unexpected breakages.

Reply:
I don't know the answer to that.

@tarcieri just switched the master branch from `0.6.0` to `0.7.0-pre`. There are a couple things that needs to go in before we can make a pre-release.

Reply:
I think I can live with depending on the trunk in the unreleased version, but I want to get some small stuff in first and release a 0.6.1 - namely, `+ ?Sized` bounds for the RNGs.

Reply:
I had to introduce a little fix to `rand_core` (https://github.com/rust-random/rand/pull/1592) but that should otherwise be rebased.

Reply:
I'm rebasing it now, and yes, I realized that a fix is necessary for `rand_core` (and made the same PR :). There's a question of whether we should propagate RNG errors, but I guess it can be done incrementally after this PR.

Reply:
I'll see if I can get it working without waiting for `rand_core`.

Reply:
I believe the only would need to introduce our own wrapper to make an implementation of `TryCryptoRng` there (reverse-`UnwrapMut` if you see what I mean).

But rand folks have been nice with me to make small patch releases: https://github.com/rust-random/rand/pull/1589
They released `0.9.1` the next day. Might be worth asking.

Reply:
added a simple stop-gap in the last commit.